modules/installation-vsphere-infrastructure.adoc

```diff
@@ -6,12 +6,12 @@
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 7.0 Update 2 or later instance that meets the requirements for the components that you use.
+You must install an {product-title} cluster on one of the following versions of a VMware vSphere instance that meets the requirements for the components that you use:
 
-[NOTE]
-====
-{product-title} version {product-version} supports VMware vSphere version 8.0.
-====
+* Version 7.0 Update 3 or later
+* Version 8.0 Update 2 or later
+
+Both of these releases support Container Storage Interface (CSI) migration, which is enabled by default on {product-title} {product-version}.
```

Comment:
@jcpowermac . OK to mention  CSI migration here?

Reply:
Just wondering if this [doc note](https://docs.openshift.com/container-platform/4.15/storage/persistent_storage/persistent-storage-vsphere.html#:~:text=For%20new%20installations%2C%20OpenShift%20Container%20Platform%204.13%20and%20later%20provides%20automatic%20migration%20for%20the) is no longer valid?

Reply:
@jsafrane can you take a look?